htdocs/compta/facture.php

```diff
@@ -3925,7 +3925,7 @@
 				|| ($object->type == Facture::TYPE_CREDIT_NOTE && empty($discount->id))
 				|| ($object->type == Facture::TYPE_DEPOSIT && empty($discount->id)))
 				&& ($object->statut == 2 || $object->statut == 3 || ($object->statut == 1 && $object->paye == 1))   // Condition ($object->statut == 1 && $object->paye == 1) should not happened but can be found due to corrupted data
-			    && $user->rights->facture->creer)				// A paid invoice (partially or completely)
+			    && $user->rights->facture->creer && $user->rights->facture->invoice_advance->reopen)				// A paid invoice (partially or completely)
```

Comment:
It's ok i just corrected that.
I'm sorry, i didn't enough know advanced user rights management before, now it's ok.
Then i restore this default right to 0 for new users, because if it's an advanced right it makes sense.